tests/test_numeric/test_eulerian_grid_ops/test_stencil_ops_2d/test_outplane_field_curl_mpi_2d.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import pytest
+from sopht.utils.precision import get_real_t, get_test_tol
+from sopht.numeric.eulerian_grid_ops.stencil_ops_2d import (
+    gen_outplane_field_curl_pyst_kernel_2d,
+)
+from sopht_mpi.utils import (
+    MPIConstruct2D,
+    MPIGhostCommunicator2D,
+    MPIFieldCommunicator2D,
+)
+from sopht_mpi.numeric.eulerian_grid_ops.stencil_ops_2d import (
+    gen_outplane_field_curl_pyst_mpi_kernel_2d,
+)
+
+
+@pytest.mark.mpi(group="MPI_stencil_ops_2d", min_size=2)
+@pytest.mark.parametrize("ghost_size", [1, 2, 3])
+@pytest.mark.parametrize("precision", ["single", "double"])
+@pytest.mark.parametrize("rank_distribution", [(1, 0), (0, 1)])
+@pytest.mark.parametrize("aspect_ratio", [(1, 1), (1, 2), (2, 1)])
+def test_mpi_outplane_field_curl_2d(
+    ghost_size, precision, rank_distribution, aspect_ratio
+):
+    n_values = 128
```

Comment:
these are too big grid sizes and eventually, tests will become slower, lets's use 32? tests are supposed to run fast.

Reply:
you can also modify grid sizes in other kernels and make 32?

Reply:
Okay let me do this in a different PR later for refactoring, I want to go over some of the docstrings and cleanup a bit where I can.

Reply:
please make an issue for this "reduce grid sizes in tests for faster CI" and then resolve this comment.

Reply:
Fixed in #26 .